Tensile/bin/TensileCreateLibrary

```diff
@@ -21,6 +21,15 @@
 ################################################################################
 # This script only gets called by CMake
 
+import google3
```

Comment:
What is this module?

Reply:
This is an unintentional local change needed to run in our environment. I'll remove it.

Reply:
Done.